.github/workflows/bow-php-action.yml

```diff
@@ -15,27 +11,28 @@ env:
 jobs:
   build-and-test:
     runs-on: ${{ matrix.os }}
-    name: PHP-${{ matrix.php }} - ${{ matrix.os }}
-
-    services:
-      mysql:
-        image: mysql:5.7
-        env:
-          MYSQL_DATABASE: test_db
-          MYSQL_ALLOW_EMPTY_PASSWORD: yes
-        ports:
-          - 3306
-        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
```

Comment:
It seems that the MySQL setup in the GitHUb action is not worked successfully.

And the test suite always encounter the `Connection Refused` issue.